windows/windows.py

```diff
@@ -0,0 +1,198 @@
+import subprocess
+import os
+import re
+import sys
+import zipfile
+
+sys.path.append("..")
+
+from utility.check_internet import check_internet
+from utility.install_package import install
+
+from path import path, var_dir, cred_dir
+
+def check_internet():
```

Comment:
if you are importing the module from utility, then why are you defining the function again